### ISSUE TITLE
Feature Custom I18n Settings.

### ISSUE THREAD

**Issue by tecnologiaterabyte** [dektrium/yii2-user#333]
Hi @dmeroff changing these simple lines in boostrap.php, we can choose between the official translation module or custom translation.

Example Code Boostrap.php (feature).

        if (!isset(\Yii::$app->i18n->translations['user*'])) { // verify config translate custom.
            $app->get('i18n')->translations['user*'] = [
                'class'    => PhpMessageSource::className(),
                'basePath' => __DIR__ . '/messages',
            ];
        }

Example Config I18n:

        'i18n' => [
            'translations' => [
                'user*' => [
                    'class' => 'yii\i18n\PhpMessageSource',
                    'basePath' => '@backend/messages',
                    'sourceLanguage' => 'en',
                    'fileMap' => array(
                        'user'=>'user.php',
                    )
                ],
            ],
        ],
Thks in advance,

**Comment by dmeroff** [comment]
Good idea! Thank you!

**Comment by kostromich** [comment]
Seems like doesn't work for me for extended Views, Controllers and Models.
Same with the original (not overridden module)
```php
'i18n' => [
            'translations' => [
                'app*' => [
                    'class' => 'yii\i18n\PhpMessageSource',
                    'basePath' => '@app/messages/app',
                    'sourceLanguage' => 'en',
                    'fileMap' => [
                        'app' => 'app.php',
                        'app/error' => 'error.php',
                    ],
                ],
                'user*' => [
                    'class' => 'yii\i18n\PhpMessageSource',
                    'basePath' => '@app/messages/user',
                    'sourceLanguage' => 'en',
                    'fileMap' => array(
                        'user'=>'user.php',
                    )
                ],
            ],
        ],
```

**Comment by kostromich** [comment]
fixed when installing v0.9.5
BTW for v0.9.5 i'v added  "yiisoft/yii2-jui": ">=2.0@dev", to require section composer.json. only 0.9.3 is available without "yii2-jui ~2.0" dependency
